src/main/java/com/google/flourbot/api/DriveClient.java

```diff
@@ -33,33 +30,34 @@
 import java.io.FileNotFoundException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
-import java.security.GeneralSecurityException;
-import java.util.Collections;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
 
-interface DriveDoc {
-  public void createService();
-  public void append();
-  public String createNew();
-}
+class DriveClient implements CloudDocClient {
 
-class MySheet implements DriveDoc {
+  static final String SPREADSHEET_SCOPE = "https://www.googleapis.com/auth/spreadsheets";
+
+  public CloudSheet getCloudSheet(String documentId) throws IOException, GeneralSecurityException {
+    DriveCloudSheet cs = new DriveCloudSheet(documentId, this.createService());
```

Comment:
no need for the variable
return new DriveCloudSheet(documentId, this.createService());